tests/parser-test-files/parse-type.txt

```diff
@@ -0,0 +1,239 @@
+================================================================================
+1 Parse type (:disable)
```

Comment:
I'll file an issue after this PR is merged to split these declare cases into individual tests